app/controllers/users_controller.rb

```diff
@@ -0,0 +1,62 @@
+
+class UsersController < ApplicationController
+
+  get '/signup' do
+    if !!session[:user_id]
+      redirect to '/tweets'
+    else
+      erb :'/users/signup'
+    end
+  end
+
+  get '/logout' do
+    if session[:user_id] !=nil
+      session.destroy
+      redirect '/login'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/login' do
+    if !!session[:user_id]
```

Comment:
if logged_in? would be better here and I would have them get redirected to `/tweets` instead of a new tweet form